src/components/projects/CreateProjectForm.vue

```diff
@@ -909,7 +924,10 @@ export default {
         lunar_dist_min: false,
         lunar_phase_max: false
       },
-      calendarBaseUrl: this.$store.state.api_endpoints.calendar_api
+      calendarBaseUrl: this.$store.state.api_endpoints.calendar_api,
+
+      cycleIsActive: true,
+      depleteIsActive: true
```

Comment:
Maybe you can think of a better place than here, but it would be nice to have some documentation on what these values represent.